sdk/storage/Azure.Storage.Blobs.ChangeFeed/src/ChangeFeedFactory.cs

```diff
@@ -117,6 +117,8 @@ internal class ChangeFeedFactory
 
             lastConsumable = jsonMetaSegment.RootElement.GetProperty("lastConsumable").GetDateTimeOffset();
 
+            jsonMetaSegment.Dispose();
```

Comment:
I think we still probably want to use a using here in case there is an exception. 
/cc @seanmcc-msft @amnguye

Reply:
updated to use try-finally